lfs/CMakeLists.txt.lfs

```diff
@@ -37,7 +37,7 @@ if(LUA51) # build against installed lua 5.1
     find_program(LUA NAMES lua lua.bat)
     add_test(NAME test COMMAND ${LUA} test.lua)
 else() # lua_sandbox build
-    set(CPACK_DEBIAN_PACKAGE_DEPENDS "${CPACK_DEBIAN_PACKAGE_DEPENDS}, luasandbox (>= 1.0.2), luasandbox-lpeg (>= 1.0.10)")
+    set(CPACK_DEBIAN_PACKAGE_DEPENDS "${CPACK_DEBIAN_PACKAGE_DEPENDS}, luasandbox (>= 1.0.2), luasandbox-lpeg (>= 1.0.10), ${PACKAGE_PREFIX}-gzfile (>= 0.0.3)")
```

Comment:
`PACKAGE_PREFIX` variable doesn't seem to be set in this context.
Maybe add:
```
set(PACKAGE_PREFIX luasandbox)
```

Reply:
sigh I just copied this from another file... replaced it with luasandbox